flintrock/scripts/install-spark.sh

```diff
@@ -2,22 +2,20 @@
 
 set -e
 
-spark_version="$1"
-distribution="$2"
+url="$1"
 
 echo "Installing Spark..."
-echo "  version: ${spark_version}"
-echo "  distribution: ${distribution}"
+echo "  from: ${url}"
 
-file="spark-${spark_version}-bin-${distribution}.tgz"
+file=$(basename ${url})
```

Comment:
Nit: We should always quote values in Bash.

I'm paranoid after having been bit too many times. 🐍